### DISCUSSION TITLE
[V3] Unable to use assertStatus after calling a method

### DISCUSSION THREAD

**Discussion by gbuckingham89** [livewire/livewire#6245] [7. Bugs (archived)]
### Livewire version

3.0.0-beta.7

### Which PHP version are you using?

PHP 8.2

### Repository URL

https://github.com/gbuckingham89/livewire-v3-assert-status

### Steps To Reproduce

1. Create a [component](https://github.com/gbuckingham89/livewire-v3-assert-status/blob/master/app/Livewire/CreatePost.php) that has a method that returns forbidden (HTTP 403)
2. Write [a test](https://github.com/gbuckingham89/livewire-v3-assert-status/blob/master/tests/Feature/Livewire/CreatePostTest.php) that asserts the component returns a 403 status code when calling a method
3. Run the test

The test fails as follows:

```
  FAILED  Tests\Feature\Livewire\CreatePostTest > returns 403 when creating post                         HttpException   

  at vendor/laravel/framework/src/Illuminate/Foundation/Application.php:1224
    1220▕         if ($code == 404) {
    1221▕             throw new NotFoundHttpException($message, null, 0, $headers);
    1222▕         }
    1223▕ 
  ➜ 1224▕         throw new HttpException($code, $message, null, $headers);
    1225▕     }
    1226▕ 
    1227▕     /**
    1228▕      * Register a terminating callback with the application.

      +2 vendor frames 
  3   app/Livewire/CreatePost.php:11
      +32 vendor frames 
  36  tests/Feature/Livewire/CreatePostTest.php:18


  Tests:    1 failed, 2 passed (3 assertions)
  Duration: 0.16s

```

### Additional information

Also occurs with the assert status helpers e.g. `assertForbidden()`.

Works ok in v2.

### Please confirm (incomplete submissions will not be addressed)

- [X] I have provided easy and step-by-step instructions to reproduce the bug.
- [X] I have provided code samples as text and NOT images.
- [X] I understand my bug report will be removed if I haven't met the criteria above.

**Comment by calebporzio** [comment]
Good call. Fixed here: https://github.com/livewire/livewire/commit/fc41d12f9146f788e9ccf98845daac8c6a69bffa

Thanks
